c/src/sbp.c

```diff
@@ -227,13 +242,45 @@ s8 sbp_remove_callback(sbp_state_t *s, sbp_msg_callbacks_node_t *node)
   return SBP_CALLBACK_ERROR;
 }
 
+/** Register a callback for ANY message type.
+ * Register a callback that is called when any
+ * valid message is received.
+ *
+ * \param msg_type Message type associated with callback
+ * \param fn       Pointer to message callback function
+ * \param context  Pointer to context for callback function
+ * \return `SBP_OK` (0) if successful, `SBP_NULL_ERROR` if fn is NULL
+ *
+ */
+s8 sbp_register_allmsg_callback(sbp_state_t *s, sbp_allmsg_callback_fn_t fn,
+                                void *context) {
+  /* Check our callback function pointer isn't NULL. */
+  if (0 == fn) {
```

Comment:
`NULL` preferred for pointer checks

Reply:
I agree, but in order to do this it requires adding a new #include that wasn't here when the original author wrote.  I thought perhaps there was a portability reason to not include stddefs.   I'm happy to follow up with a separate PR to add in NULL for null pointer and the new include, but for now I wanted to follow the existing pattern.